generic/interbench.py.data/makefile_fix.patch

```diff
@@ -0,0 +1,20 @@
+--- interbench-0.30/Makefile.orig	2006-03-03 20:33:54.000000000 -0500
++++ interbench-0.30/Makefile	2016-06-17 10:55:17.751378605 -0400
+@@ -1,10 +1,11 @@
+-CC=gcc
+-CFLAGS=-W -Wall -g -O2 -s -pipe
+-LDFLAGS=-lrt -lm
++CC ?= gcc
++CFLAGS=-W -Wall -g -lpthread -O2 -s -pipe
++LDFLAGS=-lrt -lm -lpthread
+ 
+-interbench: interbench.o hackbench.o
+-interbench.o: interbench.c
+-hackbench.o: hackbench.c
++default : 
```

Comment:
there are some empty spaces, but sure, it works... Btw I sent the file to Con so hopefully he'll include it in the next release (no response yet)